Cargo.toml

```diff
@@ -37,12 +39,12 @@ features = ["std", "derive"]
 
 [dev-dependencies]
 base64 = "0.11.0"
-sha-1 = "0.8.1"
-sha2 = "0.8.0"
 hex = "0.4.0"
 serde_test = "1.0.89"
 rand_xorshift = "0.2.0"
 pem = "0.7"
+sha-1 = "0.8.1"
+sha3 = "0.8.1"
```

Comment:
Perhaps `sha3` (and arguably `sha-1`) should be optional? I'd say it's unusual to use any SHA-3 family function in conjunction with RSA...

Reply:
these are dev dependencies

Reply:
Whoops, my bad 😅

Reply:
Whoops, my bad 😅

Reply:
Whoops, my bad 😅